dotnetv3/Cognito/README.md

```diff
@@ -22,20 +22,21 @@ the minimum permissions required to perform the task. For more information, see
 
 ### Single actions
 Code excerpts that show you how to call individual service functions.
-* [Confirm a user](scenarios/Cognito_Basics/Cognito_MVP/CognitoMethods.cs) (`ConfirmSignUpAsync`)
-* [Get a token to associate an MFA application with a user](scenarios/Cognito_Basics/Cognito_MVP/CognitoMethods.cs) (`AssociateSoftwareTokenAsync`)
-* [Get information about a user](scenarios/Cognito_Basics/Cognito_MVP/CognitoMethods.cs) (`AdminGetUserAsync`)
-* [Resend a confirmation code](scenarios/Cognito_Basics/Cognito_MVP/CognitoMethods.cs) (`ResendConfirmationCodeAsync`)
-* [Respond to an authentication challenge](scenarios/Cognito_Basics/Cognito_MVP/CognitoMethods.cs) (`RespondToAuthChallengeAsync`)
-* [Sign up a user](scenarios/Cognito_Basics/Cognito_MVP/CognitoMethods.cs) (`SignUpAsync`)
-* [Start authentication with a tracked device](scenarios/Cognito_Basics/Cognito_MVP/CognitoMethods.cs) (`InitiateAuthAsync`)
-* [Verify an MFA application with a user](scenarios/Cognito_Basics/Cognito_MVP/CognitoMethods.cs) (`VerifySoftwareTokenAsync`)
+* [Confirm a user](Actions/CofnitoWrapper.cs) (`ConfirmSignUpAsync`)
+* [Confirm an MFA Device for tracking](Actions/CognitoWrapper.cs)
```

Comment:
Device --> device

Is this missing the ending item in parentheses?

Reply:
Yes, it was. Fixed.